src/TokenAirdrop.hs

```diff
@@ -43,10 +45,28 @@ tokenAirdrop config = do
         utxos <- utxosAt config $ config.ownAddress
         let lookups = Constraints.unspentOutputs utxos
 
-        submitTx @Void config pubKeyAddressMap lookups tx
+        eTxId <- submitTx @Void config pubKeyAddressMap lookups tx
+        case eTxId of
+          Left err -> pure $ Left err
+          Right txId -> do
+            putStrLn $ "Submitted transaction successfully: " ++ show txId
```

Comment:
It might be a good idea to add something like "waiting for tx to be confirmed" to the output, so the user knows the program is still running, and the tx is not confirmed yet.

Reply:
Right after that print, the busy waiter will print "Waiting a block then checking own UTxOs...". Do you think I need more than that?

Reply:
Oh, I haven't seen that. That's enough, yeah